scale/job/management/commands/scale_post_steps.py

```diff
@@ -71,6 +76,10 @@ def handle(self, **options):
                 exit_code = NFS_EXIT_CODE
             elif isinstance(ex, IOError):
                 exit_code = IO_EXIT_CODE
+            elif isinstance(ex, InvalidResultsManifest):
+                exit_code = IV_MF_CODE
+            elif isinstance(ex, ResultsManifestAndInterfaceDontMatch):
```

Comment:
We want to get rid of the ResultsManifestAndInterfaceDontMatch exception as it is confusing and doesn't tell the user what the problem is. We need a MissingRequiredOutput exception instead. Also improve the logging so that if a required output is not provided by the manifest, the details (which output was required, etc) are logged. Also create any additional exceptions/logging needed to cover all error cases currently handled by ResultsManifestAndInterfaceDontMatch (such as output type mismatches).